src/minter-did/src/id256.rs

```diff
@@ -100,6 +104,45 @@ impl Id256 {
         bytes[19] = NO_TO_TOKEN_MARK;
         H160::from_slice(&bytes)
     }
+
+    /// Convert BTC rune id into id256 type.
+    ///
+    /// Runes are identified by block_id/transaction index in which the rune was etched.
+    ///
+    /// If `runes` feature is enabled, `rune_id.into()` can be used with the same effect.
+    pub fn from_rune_id(block_id: u64, tx_id: u32) -> Self {
```

Comment:
Do we need this fn? The name is confusing and not idiomatic; in fact, it is called `from_rune_id` but it does not accept a `RuneId` as an argument. Same for the `to_rune_id` fn that does not produce a `RuneId`.

Reply:
Good point. On the other hand these functions can be useful for other things like NFT inscriptions and BRC20 tokens. So I renamed them into `from_btc_tx_index` and `to_btc_tx_index`. Accordingly, I've changed the mark const name into `BTC_TX_MARK` to make it more general.

Now, if the `runes` feature is disabled, there's no mentioning of runes at all, which is a good thing.